analyses/focal-cn-file-preparation/01-prepare-cn-file.R

```diff
@@ -102,18 +112,20 @@ process_annotate_overlaps <- function(cnv_df,
                                           keys = ensembl,
                                           column = "SYMBOL",
                                           keytype = "ENSEMBL"),
-      cytoband = AnnotationDbi::mapIds(org.Hs.eg.db::org.Hs.eg.db,
+      cytoband_org = AnnotationDbi::mapIds(org.Hs.eg.db::org.Hs.eg.db,
```

Comment:
Are we just using the Cytoband from UCSC? If so, why do we need this?

Reply:
You are correct, I kept it for documentation purposes but will remove it in the upcoming commit.